framework/requirements-dev.txt

```diff
@@ -6,5 +6,5 @@ pytest==7.3.1
 pytest-aiohttp==1.0.4
 pytest-asyncio==0.18.1
 pytest-html==2.1.1
-pytest-trio==0.7.0
+pytest-trio==0.8.0
 tavern==1.23.5
```

Comment:
Please add a line at the end of the file. Some editors can be configured to do this automatically ([VSCode](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)), I would suggest enabling that option if available.